Dockerfile

```diff
@@ -11,12 +11,11 @@ ENV TERESAK8S_HOST $TERESAK8S_HOST
 ENV TERESAK8S_USERNAME $TERESAK8S_USERNAME
 ENV TERESAK8S_PASSWORD $TERESAK8S_PASSWORD
 ENV TERESAK8S_INSECURE $TERESAK8S_INSECURE
-ENV TERESABUILDER_STORAGE_AWS_KEY $TERESABUILDER_STORAGE_AWS_KEY
-ENV TERESABUILDER_STORAGE_AWS_SECRET $TERESABUILDER_STORAGE_AWS_SECRET
-ENV TERESABUILDER_STORAGE_AWS_REGION $TERESABUILDER_STORAGE_AWS_REGION
-ENV TERESABUILDER_STORAGE_AWS_BUCKET $TERESABUILDER_STORAGE_AWS_BUCKET
-ENV TERESABUILDER_WAIT_POD_TIMEOUT $TERESABUILDER_WAIT_POD_TIMEOUT
-ENV TERESABUILDER_WAIT_LB_TIMEOUT $TERESABUILDER_WAIT_LB_TIMEOUT
+ENV TERESAFILESTORAGE_TYPE $TERESAFILESTORAGE_TYPE
+ENV TERESAFILESTORAGE_AWS_KEY $TERESAFILESTORAGE_AWS_KEY
+ENV TERESAFILESTORAGE_AWS_SECRET $TERESAFILESTORAGE_AWS_SECRET
+ENV TERESAFILESTORAGE_AWS_REGION $TERESAFILESTORAGE_AWS_REGION
+ENV TERESAFILESTORAGE_AWS_BUCKET $TERESAFILESTORAGE_AWS_BUCKET
```

Comment:
maybe for another time but, I think this environment variables should be called: 
```
TERESAFILESTORAGE_AUTH_KEY
TERESAFILESTORAGE_AUTH_SECRET
TERESAFILESTORAGE_BUCKET
TERESAFILESTORAGE_REGION
```
Because I think the future idea is to make Teresa works with another file storages (e.g. minio) right?
Every `_AWS_` name looks like something that only runs on AWS infrastructure. What do you think? (I can make this change after the EPIC MERGE 😂 )